### PR TITLE
Add an example of a tiny image -- 1x1 pixel

### PR DESCRIPTION
**Pull request by hmaarrfk**
I swear when i started to use PyGFX this was not possible. Specifically, 1x1 pixel images would return a strange result.

<details><summary>old image</summary>

image
</details>

image

I know i'm pixel peeping, but there seems to be a "border" around my image when I zoom in:
image

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Hmm even failing locally with:

```
PYGFX_WGPU_ADAPTER_NAME=llvmpipe pytest examples/tests/test_examples.py::test_examples_compare[validate_tiny_image] --verbose --regenerate-screenshots
```

File: examples/validation/validate_tiny_image.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+for i in [
+    32,
+    16,
+    8,
+    4,
+    2,
+    1,
+]:
+    image_rgb = np.zeros((i, i, 3), dtype=np.uint8)
+    image_rgb[...] = np.random.randint(0, 256, image_rgb.shape)
```
Comment (hmaarrfk) [inline examples/validation/validate_tiny_image.py:44]: need to refactor to use a random seed.

**Comment by Korijn** [comment]
I love this test case

**Comment by claydugo** [comment]
> I swear when i started to use PyGFX this was not possible.

It failed June 2023 for sure because of [this line](https://github.com/pygfx/pygfx/blob/9438fc51aef71bf571ba/pygfx/resources/_texture.py#L252).

**Comment by hmaarrfk** [comment]
> It failed June 2023 for sure because of [this line](https://github.com/pygfx/pygfx/blob/9438fc51aef71bf571ba/pygfx/resources/_texture.py#L252).

such a detective 🕵🏾

**Review by Korijn — APPROVED** [review]

**Pull request merged**